pkg/node/node.go

```diff
@@ -802,6 +802,32 @@ func NewBee(
 		}
 	)
 
+	if !o.Resync && !batchStoreExists && (networkID == mainnetNetworkID) {
```

Comment:
To be able to test, we should have or different build, or we could use `use-postage-snapshot` flag (set default to be true), but in this case we should remove the `bootstrapNode` logic.